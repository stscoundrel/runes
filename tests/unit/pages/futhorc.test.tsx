import ReactDOM from 'react-dom'
import Futhorc from 'pages/futhorc'
import renderer from 'react-test-renderer'

describe('Futhorc runic keyboard page', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Futhorc />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Futhorc />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
