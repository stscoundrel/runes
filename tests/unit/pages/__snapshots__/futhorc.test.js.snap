// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Futhorc runic keyboard page Matches snapshot 1`] = `
Array [
  <main
    className="container"
  >
    <section>
      <header>
        <h1>
          Futhorc
        </h1>
        <p>
          Translate runes to text
        </p>
      </header>
      <p>
        
      </p>
      <button
        onClick={[Function]}
      >
        ᚠ
      </button>
      <button
        onClick={[Function]}
      >
        ᚢ
      </button>
      <button
        onClick={[Function]}
      >
        ᚦ
      </button>
      <button
        onClick={[Function]}
      >
        ᚩ
      </button>
      <button
        onClick={[Function]}
      >
        ᚱ
      </button>
      <button
        onClick={[Function]}
      >
        ᚳ
      </button>
      <button
        onClick={[Function]}
      >
        ᚷ
      </button>
      <button
        onClick={[Function]}
      >
        ᚹ
      </button>
      <button
        onClick={[Function]}
      >
        ᚻ
      </button>
      <button
        onClick={[Function]}
      >
        ᚾ
      </button>
      <button
        onClick={[Function]}
      >
        ᛁ
      </button>
      <button
        onClick={[Function]}
      >
        ᛡ
      </button>
      <button
        onClick={[Function]}
      >
        ᛄ
      </button>
      <button
        onClick={[Function]}
      >
        ᛇ
      </button>
      <button
        onClick={[Function]}
      >
        ᛈ
      </button>
      <button
        onClick={[Function]}
      >
        ᛉ
      </button>
      <button
        onClick={[Function]}
      >
        ᛋ
      </button>
      <button
        onClick={[Function]}
      >
        ᚴ
      </button>
      <button
        onClick={[Function]}
      >
        ᛏ
      </button>
      <button
        onClick={[Function]}
      >
        ᛒ
      </button>
      <button
        onClick={[Function]}
      >
        ᛖ
      </button>
      <button
        onClick={[Function]}
      >
        ᛗ
      </button>
      <button
        onClick={[Function]}
      >
        ᛚ
      </button>
      <button
        onClick={[Function]}
      >
        ᛝ
      </button>
      <button
        onClick={[Function]}
      >
        ᛟ
      </button>
      <button
        onClick={[Function]}
      >
        ᛞ
      </button>
      <button
        onClick={[Function]}
      >
        ᚪ
      </button>
      <button
        onClick={[Function]}
      >
        ᚫ
      </button>
      <button
        onClick={[Function]}
      >
        ᚣ
      </button>
      <button
        onClick={[Function]}
      >
        ᛠ
      </button>
      <button
        onClick={[Function]}
      >
        :
      </button>
    </section>
  </main>,
  <footer
    className="section"
  >
    <div
      className="container text-center"
    >
      <small
        className="copyright"
      >
        Copyright © 
        2021
        <a
          href="https://github.com/stscoundrel"
          rel="noopener noreferrer"
          target="_blank"
        >
          Sampo Silvennoinen / StScoundrel
        </a>
      </small>
    </div>
  </footer>,
]
`;
